pages.es/linux/vnstati.md

```diff
@@ -0,0 +1,20 @@
+# vnstati
+
+> Compatibilidad de salida a imagen PNG para vnStat.
```

Comment:
```suggestion
> Genera imágenes PNG compatibles con vnStat.
```

Reply:
I think "Soporte de salida de imágenes PNG para vnStat." will be better

Reply:
I disagree, AFAIK, your sentence is not in an imperative mood.

Reply:
I am agree with @tricantivu point of view.

Reply:
On "PNG image output support for vnStat."  the part "PNG image output support" acts as noun and "for vnStat" is complement, there is not really a verb to make it imperative.
Maybe "Generador de imágenes PNG compatible con vnStat.", but again is not imperative.

Reply:
I get that, however, rewriting the description as suggested might be clearer. I personally, do not like how sentences sound with `salida`  (e.g. `Genera salida`) nor other alternatives such as `Imprime datos`, `Imprime imágenes`.